src/components/pages/ContactPage.jsx

```diff
@@ -79,7 +79,7 @@ function ContactUsFormComponent() {
             <label>
               <Field type="checkbox" name="toggle" />&nbsp;
               All communication must adhere to our&nbsp;
-              <a className={formStyleClasses.hyperlinks} href="/pages">
+              <a className={formStyleClasses.hyperlinks} href="/codeofconduct">
```

Comment:
@reshamas, we will want to use a React Router Link here.

You just need to update line 9 from 

```JavaScript
import { useNavigate } from 'react-router-dom';
```
to
```JavaScript
import { Link, useNavigate } from 'react-router-dom';
```
and update this line to

```JavaScript
<Link to="/codeofconduct" className={formStyleClasses.hyperlinks}>
  Code of Conduct
</Link>*
```